exercises/F-strings-methods/exercise.js

```diff
@@ -1,3 +1,4 @@
 // Start by creating a variable `message`
-
-console.log(message);
+var names = " Esam ";
+var nameLength = names.trim();
+console.log(nameLength);
```

Comment:
Good use of .trim. You also need to use .length to get the number of letters in the name.